src/Utils/utils.ts

```diff
@@ -32,10 +33,21 @@ export const formatTimeShort = (time: Time) => {
 };
 
 export const relativeDate = (date: DateLike, withoutSuffix = false) => {
+  const { t } = useTranslation();
```

Comment:
violates rules of hooks! you can however simply import `t` btw. Since `t` would be invoked lazily inside the function, it'd work as expected as well.
```suggestion
```

Reply:
https://react.dev/reference/rules/rules-of-hooks